src/fixtures/activity.js

```diff
@@ -2,7 +2,8 @@ const activity = {
   id: '8437fa68-8e6b-11e8-a05c-9801a7ae0330',
   category: 'Participating in a tech event',
   date: 'November 3, 2017',
-  description: 'Mentored teens how to code. (DBC 2016 at Redemption camp). Jim Shelton of ChanZuckerberginitiative sits down with Andela fellows',
+  activityDate: 'November 3, 2017',
+  description: 'Mentored teens how to code. (DBC 2016 at Redemption camp).Jim Shelton of ChanZuckerberginitiative sits down with Andela fellows',
```

Comment:
@puritybirir You could remove the `date` key since it's no more needed.

Reply:
On it